docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -89,15 +89,19 @@ that don’t contain a results field are not included in the {reganalysis}.
   (Optional, object) You can specify both `includes` and/or `excludes` patterns. 
   If `analyzed_fields` is not set, only the relevant fields will be included. 
   For example, all the numeric fields for {oldetection}. For the supported field 
-  types, see <<ml-put-dfanalytics-supported-fields>>.
+  types, see <<ml-put-dfanalytics-supported-fields>>. If you specify fields – 
+  either in `includes` or in `excludes` – that have a data type that is not 
```

Comment:
That is not accurate. Specifying a field that has an unsupported type in `excludes` is not causing error. Have you observed otherwise?

Reply:
@szabosteve Has confirmed my comment is not the case.